edx-platform/pearson-pathways-theme/lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -153,9 +153,19 @@ <h3 class="course-title" id="course-title-${enrollment.course_id}">
                     </span>
                 % else:
                     % if isinstance(course_date, six.string_types):
-                        <span class="info-date-block">${container_string.format(date=course_date)}</span>
+                      <span class="info-date-block">${container_string.format(date=course_date)}</span>
                     % elif course_date is not None:
-                        <span class="info-date-block localized-datetime" data-language="${user_language}" data-timezone="${user_timezone}" data-datetime="${course_date.strftime('%Y-%m-%dT%H:%M:%S%z')}" data-format=${format} data-string="${container_string}"></span>
+                      <%
+                        # Formatting the date string like this: Started - <b>{date}</b>
+                        container_string = Text(container_string).format(
+                          date=Text('{bold_start}{date}{bold_end}').format(
+                            bold_start=HTML('<b>'),
+                            date=course_date.strftime('%b %-d, %Y'),
```

Comment:
@Squirrel18, just to know, what is the difference?
```suggestion
                            date=course_date.strftime('%b %d, %Y'),
```

Reply:
@ivanvgh Removes the zero-padded value. https://stackoverflow.com/questions/904928/python-strftime-date-without-leading-0

Reply:
Thanks.